src/quickstart/provider/aws/boot.md

```diff
@@ -1,263 +1,364 @@
-$ tar -xvzf kubernetes.tar.gz
-$ cd kubernetes
-$ ls
-LICENSES    README.md   Vagrantfile client/      cluster/     docs/        examples/    federation/  server/      third_party/ version
+
+Note the *SecretAccessKey* and *AccessKeyID* so you can enter them in the following commands
+
+```bash
+aws configure # Input your credentials here
+aws iam list-users
 ```
 
-## Configure the Kubernetes Environment
 
-Before calling the Kubernetes setup scripts, we need to change a few defaults so that Deis Workflow works best. Type
-each of these commands into your terminal application before calling `kube-up.sh`.
+#### Configure DNS
+
+In order to build a Kubernetes cluster with `kops`, we need to prepare
+somewhere to build the required DNS records.  There are three scenarios
+below and you should choose the one that most closely matches your AWS
+situation.
+
+#### Scenario 1a: A Domain purchased/hosted via AWS
+
+If you bought your domain with AWS, then you should already have a hosted zone
+in Route53.  If you plan to use this domain then no more work is needed.
+
+In this example you own `example.com` and your records for Kubernetes would
+look like `etcd-us-east-1c.internal.clustername.example.com`
+
+You can now skip to [testing your DNS setup](#testing-your-dns-setup)
+
+#### Scenario 1b: A subdomain under a domain purchased/hosted via AWS
 
-Next, pick the AWS Availability Zone you would like to use. The boot script will create a new VPC in that region.
+In this scenario you want to contain all kubernetes records under a subdomain
+of a domain you host in Route53.  This requires creating a second hosted zone
+in route53, and then setting up route delegation to the new zone.
 
+In this example you own `example.com` and your records for Kubernetes would
+look like `etcd-us-east-1c.internal.clustername.kubernetes.example.com`
+
+This is copying the NS servers of your **SUBDOMAIN** up to the **PARENT**
+domain in Route53.  To do this you should:
+
+
+```bash
+ID=$(uuidgen) && aws route53 create-hosted-zone --name subdomain.example.com --caller-reference $ID | jq .DelegationSet.NameServers
 ```
-export KUBE_AWS_ZONE=us-west-1c
-export KUBERNETES_PROVIDER=aws
+
+* Note your **PARENT** hosted zone id
+
+```bash
+# Note: This example assumes you have jq installed locally.
+aws route53 list-hosted-zones | jq '.HostedZones[] | select(.Name=="example.com.") | .Id'
 ```
 
-For evaluation, we find that the t2 instance classes are a reasonable bang for the buck. Do note that the t2 class does
-track CPU credits. Performance of your evaluation cluster may be impacted when you exhaust the CPU credit limit. Select
-your instance sizes and worker count.
+* Create a new JSON file with your values (`subdomain.json`)
+
+Note: The NS values here are for the **SUBDOMAIN**
 
 ```
-export MASTER_SIZE=t2.medium
-export NODE_SIZE=t2.large
-export NUM_NODES=2
-export NODE_ROOT_DISK_SIZE=100
+{
```

Comment:
There is other docs in the wild that suggest you do it this way, I would vote to keep them uniform.